### DISCUSSION TITLE
How can I get this stuffs?

### DISCUSSION THREAD

**Discussion by profLeonardoRocha** [community/community#37256] [GitHub Education]
Hi guys, I hope you all safe. Someone knows how can I get this stuff (see the file)? I'm teacher and would like to teach use github to my students and it'll be great to give them these printed materials.
![foto](https://user-images.githubusercontent.com/86802310/197870998-a3be77cd-78bc-41c1-93b7-df5d20a8cc23.jpeg)

Thanks a lot guys.

**Comment by melvincwng** [comment]
Hi @LeonardoRochaMarista, you can apply for the GitHub Teachers Program over [here](https://education.github.com/teachers) to gain access to a variety of tools/cheatsheets/resources for your students.

Hope this answers your question 😊

**Reply by profLeonardoRocha** [reply]
Hi @melvincwng thanks for the answer. I'll see how I can order the printed material.
